packages/sw-background-sync-queue/demo/index.html

```diff
@@ -0,0 +1,28 @@
+<html>
+	<head></head>
+	<body>
+		<div>Hello dear</div>
+		<button onclick="putReqInQ('1')">send req</button>
+		<button onclick="putReqInQ('2')">send req</button>
+		<button onclick="putReqInQ('3')">send req</button>
+		<button onclick="putReqInQ('4')">send req</button>
+		<button onclick="putReqInQ('5')">send req</button>
+		<script>
+			var _reg;
```

Comment:
@jeffposnick @addyosmani i'll be droppping this demo all together to make a more presentable one, this was just to see if the POC was working.
request you to omit the demo portion for now